src/classes/ACCT_IndividualAccounts_TDTM.cls

```diff
@@ -809,7 +815,7 @@ public class ACCT_IndividualAccounts_TDTM extends TDTM_Runnable {
         if (ADDR_Addresses_UTIL.isStateCountryPicklistsEnabled)
             strSoql += 'MailingCountryCode, MailingStateCode, OtherCountryCode, OtherStateCode, ';
 
-        strSoql += 'Phone, Fax from Contacts)';
+        strSoql += 'Phone, Fax from Contacts WHERE Exclude_from_Household_Name__c != true)';
```

Comment:
I fixed a minor spacing issue here where there wasn't a space between ...Household_Name__c and != 

No sense turning that back to you when I can just do it directly here :)